lab_1/main.py

```diff
@@ -74,7 +117,12 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
+    if isinstance(unknown_profile, dict) == isinstance(profile_1, dict) == isinstance(profile_2, dict) == isinstance(top_n, int)== True:
```

Comment:
There should be a check for the input arguments at the top of the function - after the documentation string. Remove nesting here and extract the check into a separate `if` block earlier.